pages/index.tsx

```diff
@@ -50,8 +52,12 @@ export default function Index() {
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const [selectedIndex, setSelectedIndex] = useState(0);
   const { address: collAddress } = Collateral.useContainer();
+  const { address: tokenAddress } = Token.useContainer();
   const { contract: weth } = WethContract.useContainer();
-  const { isYieldToken } = Balancer.useContainer();
```

Comment:
Removing the Balancer.useContainer() call here makes sure that the dApp is not making any Balancer subgraph queries at the `index.tsx` level